### PR TITLE
BUILD: Refactor wolfssl submodule

### PR DESCRIPTION
**Pull request by p01arst0rm**
moved wolfssl files into separate directory.

### CODE REVIEW

**Comment by p01arst0rm** [comment]
cc @hcorion

File: 3rdparty/wolfssl/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,51 @@
+
```
Comment (Megamouse) [inline 3rdparty/wolfssl/CMakeLists.txt:1]: why are there always empty lines at the top of your files?

Comment (p01arst0rm) [reply 3rdparty/wolfssl/CMakeLists.txt:1]: auto padding maybe? :*

**Review by hcorion — APPROVED** [review]
Looks fine 👍

**Pull request merged**